xml/System.Diagnostics/Activity.xml

```diff
@@ -639,8 +655,9 @@ An ID has a hierarchical structure: `root-id.id1_id2.id3_`. The ID is generated
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Indicates if the this activity should be populated with all the propagation information, as well as all the other properties such as links, tags and events.</summary>
+        <value>
+          <see langword="true" /> if the activity should be populated; <see langword="false" /> otherwise.</value>
         <remarks>To be added.</remarks>
```

Comment:
><remarks>To be added.</remarks> [](start = 8, length = 31)

could you please add the remark:

The best practice for the library authors is not try to set this property. This property is automatically set when creating the Activity. The setter is useful only in the case of Activity objects created using Activity constructor and want to override the default value. Activity objects created by ActivitySource.StartActivity is automatically set this property to the right value.